setup.py

```
@@ -153,9 +153,9 @@ def parse_requirements_for_install_requires(requirements_text):
 			str("""Topic :: Software Development :: Libraries :: Python Modules"""),
 			str("""Topic :: System :: Networking""")
 		]
-	except Exception as e:
+	except Exception as _cause:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)